test/CloudNative.CloudEvents.UnitTests/Kafka/KafkaTest.cs

```diff
@@ -78,7 +78,6 @@ public void KafkaStructuredMessageTest()
             AssertTimestampsEqual("2018-04-05T17:31:00Z", receivedCloudEvent.Time!.Value);
             Assert.Equal(MediaTypeNames.Text.Xml, receivedCloudEvent.DataContentType);
             Assert.Equal("<much wow=\"xml\"/>", receivedCloudEvent.Data);
-
```

Comment:
I'm not sure why this line has been removed, but there's a blank line in the equivalent assertion that you've added at (new) line 128.

It would be good if this PR *only* contained changes that were actually relevant to the feature.

Reply:
This conversation has been resolved, but I still see the change... again, I think the PR would be best if it were *just* changes related to the feature. (Other formatting changes can come in a separate PR, of course.)